### ISSUE TITLE
Cannot install on armv7 (0.9.0)

### ISSUE THREAD

**Issue by echo-bravo-yahoo** [pimalaya/himalaya#500]
```
pi@REDACTED:~/$ cargo install himalaya
    Updating crates.io index
  Downloaded himalaya v0.9.0
  Downloaded 1 crate (89.4 KB) in 0.22s
  Installing himalaya v0.9.0
    Updating crates.io index
error: failed to compile `himalaya v0.9.0`, intermediate artifacts can be found at `/tmp/cargo-installx8p5Eq`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.

Caused by:
  failed to select a version for the requirement `keyring-lib = "=0.1.0"`
```

**Comment by soywod** [comment]
Main repositories are not up-to-date (see README, installation section). Could you try to install from git instead? Meanwhile, I will check if I can add armv4 pre-build binaries to CI.

**Comment by soywod** [comment]
Which targets does it refer to? `armv7-unknown-linux-gnueabi`?

**Comment by soywod** [comment]
Armv6 and v7 are now available thanks to #506, feel free to reopen in case of issue with the binary.
